tests/foreman/ui/test_provisioningtemplate.py

```diff
@@ -80,6 +80,35 @@ def test_positive_clone(session, clone_setup):
         assert set(clone_setup['os_list']) == {f'{os.name} {os.major}' for os in assigned_oses}
 
 
+@pytest.mark.tier2
+def test_positive_clone_locked(session, target_sat):
```

Comment:
Do you think the `caseLevel` and the `caseImportance` for this test case is same as defined at module level? If not please change is accordingly.

At least I doubt on caseLevel to be acceptance here, I think it should be `Integration` or something else like `system` etc.

Reply:
yes, updated